### DISCUSSION TITLE
UM 13.6 and Jules 7.6 Releases

### DISCUSSION THREAD

**Discussion by james-bruten-mo** [MetOffice/simulation-systems#216] [Announcements]
Dear all,

I'm happy to announce that UM vn13.6 and JULES vn7.6 have now been released. Many thanks to all developers and reviewers for their efforts and contributions, and thank you for your patience over a longer than usual closed release period.

Documentation and Release Notes will follow shortly as will prebuilds for meto users.

Many thanks,

James, on behalf of the SSD Team

**Comment by Adrian-Lock** [comment]
Not sure if I've done something wrong but is vn13.6 defined yet for the UM documentation papers?  I just tried the following:

$  fcm bc --type=DEV --ticket=7705 bm_cflmax_in_cff_doc fcm:um_doc.x_tr@vn13.6
[FAIL] svn info --xml fcm:um_doc.x_tr@vn13.6 # rc=1
[FAIL] svn: E205000: Try 'svn help info' for more information
[FAIL] svn: E205000: Syntax error parsing peg revision 'vn13.6'

**Reply by jennyhickson** [reply]
UMDP release is still pending. I should get to it in the next couple of days. Sorry for the delay

**Reply by jennyhickson** [reply]
Documentation release is now complete so you should be able to create vn13.6 um_doc branches now. Thanks
